### ISSUE TITLE
Distinguishing plaintext fields with and without line breaks

### ISSUE THREAD

**Issue by missmatsuko** [craftcms/cms#4030]
### Description
It's possible to create plaintext fields in Craft CMS with and without line breaks. With line breaks enabled, it's possible to set a number for initial rows. 

When line breaks are enabled, and initial rows is set to 1, the plaintext field appears identical to a plaintext field without line breaks enabled.

For content entry, it's important to know whether the field will accept line breaks or not. If I try to enter a new line into a plaintext field without line breaks, this submits the form and saves the entry. It could be really unexpected and end up publishing a partially-completed entry.

I'm not sure if it should be the responsibility of the dev to set an initial rows of greater than 1 to indicate that the field accepts more than 1 line of text. Personally, I kind of like having that option for fields where it should only have 1 line of text *most* of the time in order to save a bit of space and to encourage authors to keep it short.

Another option I was thinking of was to have the draggable resize corner on the bottom right like native HTML textareas (but only allow resizing vertically, not horizontally): <https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea>

### Steps to reproduce
1. Create 2 plaintext fields: 1 with line breaks disabled, 1 with line breaks enabled and initial rows set to 1.
2. Try entering text with line breaks into these fields.

### Additional info

- Craft version: 3.1.15
- PHP version:
- Database driver & version:
- Plugins & versions:

**Comment by brandonkelly** [comment]
> Another option I was thinking of was to have the draggable resize corner on the bottom right like native HTML textareas (but only allow resizing vertically, not horizontally)

Good idea! Done for the next release.
